frontend/utils.js

```diff
@@ -0,0 +1,91 @@
+function createBox(id){
+    const anchor = document.createElement("a");
+    anchor.classList.add("todo");
+    anchor.classList.add("hover");
+
+    anchor.addEventListener('click', e => {
+        location.href = `/edit/${id}?`
+    })
+
+    return anchor
+};
+
+function createContent(tasks, color, title, fontSize){
+    let content = document.createElement("div");
```

Comment:
You can use const since the variable is not changing, adding or removing elements from objects/arrays doesn't mutate the variable itself